packages/node/src/evm/templates/template-application.test.ts

```diff
@@ -12,7 +12,7 @@ describe('mergeApiCallsWithTemplates', () => {
 
   it('merges the template into the API call', () => {
     const apiCall = fixtures.requests.buildApiCall({
-      airnodeAddress: null,
+      airnodeAddress: 'templateAirnode-0',
```

Comment:
seems like a strange address to me

Reply:
Yeah I thought the same but I was trying to be consistent with other values in the same file. We don't really care in this tests if it's a valid address. All we care about is that it is not null but I'll change it to an address just because it makes more sense.